docs/spectrum/fitting.rst

```diff
@@ -17,75 +17,73 @@ Getting Started
 ===============
 
 The following example shows how to fit a power law simultaneously to two
-simulated crab runs using the `~gammapy.spectrum.SpectrumFit` class.
+simulated crab runs using the `~gammapy.utils.fitting.Fit` class.
 
 .. code-block:: python
 
-    from gammapy.spectrum import SpectrumObservation, SpectrumObservationList, SpectrumFit
+    from gammapy.spectrum import SpectrumDatasetOnOff
+    from gammapy.utils.fitting import Fit
     from gammapy.spectrum.models import PowerLaw
     import matplotlib.pyplot as plt
 
     path = "$GAMMAPY_DATA/joint-crab/spectra/hess/"
-    obs1 = SpectrumObservation.read(path + "pha_obs23523.fits")
-    obs2 = SpectrumObservation.read(path + "pha_obs23592.fits")
-    obs_list = SpectrumObservationList([obs1, obs2])
+    obs_1 = SpectrumDatasetOnOff.read(path + "pha_obs23523.fits")
+    obs_2 = SpectrumDatasetOnOff.read(path + "pha_obs23592.fits")
 
     model = PowerLaw(
         index=2,
         amplitude='1e-12  cm-2 s-1 TeV-1',
         reference='1 TeV',
     )
 
-    fit = SpectrumFit(obs_list=obs_list, model=model, stat='wstat')
+    obs_1.model = model
+    obs_2.model = model
+
+    fit = Fit([obs_1, obs_2])
     result = fit.run()
 
-You can check the fit results by looking at
-`~gammapy.spectrum.SpectrumFitResult` that is attached to the
-`~gammapy.spectrum.SpectrumFit` for each observation.
+model.parameters.covariance = result.parameters.covariance
+You can check the fit results by looking at the result and model object:
 
 .. code-block:: python
 
     >>> print(result)
-    FitResult
+
+        OptimizeResult
 
         backend    : minuit
         method     : minuit
         success    : True
-        nfev       : 99
-        total stat : 63.00
+        nfev       : 115
+        total stat : 65.36
         message    : Optimization terminated successfully.
 
-    >>> print(fit.result[0])
 
-    Fit result info
-    ---------------
-    Model: PowerLaw
+    >>> print(model)
 
-    Parameters:
+        PowerLaw
 
-           name     value     error         unit         min    max
-        --------- --------- --------- --------------- --------- ---
-            index 2.761e+00 1.094e-01                       nan nan
-        amplitude 5.118e-11 4.849e-12 1 / (cm2 s TeV)       nan nan
-        reference 1.000e+00 0.000e+00             TeV 0.000e+00 nan
```

Comment:
Note, that the numbers here changed for the sherpa example (see below) equivalently. So the different numbers are likely to a change in the input data and not because of the `SpectrumFit` -> `Fit` transition.